vnext/ReactWindowsCore/ReactWindowsCore.vcxproj

```diff
@@ -98,6 +98,14 @@
       </AdditionalIncludeDirectories>
       <AdditionalIncludeDirectories Condition="'$(USE_HERMES)'=='true'">$(HERMES_Package)\installed\$(VcpkgTriplet)\include\;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
       <PreprocessToFile>false</PreprocessToFile>
+      <AdditionalOptions Condition="'$(Configuration)|$(Platform)'=='Debug|ARM'">/await /permissive- %(AdditionalOptions)</AdditionalOptions>
```

Comment:
>>/await [](start = 80, length = 7)

Could you make it one line without conditions per configuration?

<AdditionalOptions >/await %(AdditionalOptions)</AdditionalOptions>